common/components/ui/Modal/index.scss

```diff
@@ -45,6 +34,17 @@ $m-anim-speed: 400ms;
   box-shadow: 0px 8px 10px -5px rgba(0, 0, 0, 0.2), 0px 16px 24px 2px rgba(0, 0, 0, 0.14),
     0px 6px 30px 5px rgba(0, 0, 0, 0.12);
 
+  &-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(#000, 0.54);
+    z-index: $zindex-modal-background;
+    display: block;
+  }
```

Comment:
This element isn't a child of `.Modal`, which is why it had its own classname.

Reply:
Yup, totally missed that, ty for pointing it out.